builder/bundles.go

```diff
@@ -499,9 +542,28 @@ func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *b
 			return err
 		}
 
-		args := merge(packagerCmd, "--installroot="+baseDir, "install")
-		args = append(args, rpmList...)
-		_, err = helpers.RunCommandOutputEnv(args[0], args[1:], []string{"LC_ALL=en_US.UTF-8"})
+		// check later if download can be done concurrently
```

Comment:
If this comment is indicating future work, maybe prepend "TODO: " to it so that we don't overlook it. And I think it should be above the line
`_, err := downloadRpms(packagerCmd, []string{"filesystem"}, chrootDir, 1)`